docs/dump_javadump.md

```diff
@@ -24,13 +24,31 @@
 
 # Java dump
 
-Java dumps, sometimes referred to as *Java cores*, are produced when the VM ends unexpectedly because of an operating system signal, `OutOfMemoryError` exception, or a user-initiated keystroke combination. You can also generate a Java dump by calling the Dump API programmatically from your application or specifying the `-Xdump:java` option on the command line.
+Java dumps, sometimes referred to as *Java cores*, are produced when the VM ends unexpectedly because of an operating system signal, `OutOfMemoryError`, or a user-initiated keystroke combination. You can also generate a Java dump by calling the Dump API programmatically from your application or specifying the `-Xdump:java` option on the command line.
+
+If your Java application crashes or hangs, Java dumps can provide useful information to help you diagnose the root cause.
+
+- If your application crashes, Javadumps are generated automatically for the following types of failure:
+    - the VM receives an unexpected signal or an assertion failure
+    - the VM runs out of memory
+- If your application hangs, you can trigger the generation of a Java dump by sending a SIGSEGV signal (`kill -3`) to the VM.
+
+    <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** On Windows, if you started the VM in a console window you can force the VM to produce a Java dump in response to a SIGBREAK signal
+    (Ctrl-Break keyboard combination). If you didn't start in a console window there is no equivalent to a Linux `kill` command on Windows for sending signals.
+    The only option here is to trigger a full system dump by finding the VM process in the **Processes** tab of the Windows Task Manager and clicking
+    **Create dump file**.
+
+To help you understand how a Java dump can help you with problem diagnosis, we've included a few scenarios at the end of the topic that might help you interpret the data:
+
+- [A crash caused by a general protection fault (gpf)](#general-protection-fault)
+- [A Java heap OutOfMemoryError (OOM)](#java-outofmemoryerror)
```

Comment:
It would be really good to add using javacores to diagnose a hang.  Most hangs aren't really hangs and using multiple javacores over a small time period can show that.  This can be added in a later update

Reply:
Hangs is on the list, as is native OOM and deadlocks. 

I just have one core file for hangs, which doesn't sound like it covers your scenario .. but we can add multiples.

Reply:
The Hang example I created we have is for a real Hang. Specifically, in the example there is one thread holding a lock that is blocking all the other worker threads.

Providing examples for situations that look like hangs, but aren't actually hangs, is a much larger topic.

Reply:
I think the important thing is to explain the technique of using multiple javacores to determine if an application is progressing or not - pointing out the relevant parts of the data to check.  

But I agree, it can be a separate task.  @SueChaplain Is there an issue opened with the list of future topics to cover?

Reply:
I'm going to get this PR merged tomorrow as it will be getting stale. I don't think there is anything in here that is wrong, and we can discuss further scenarios and tweaks in another issue. @DanHeidinga I'll make sure I involve you and @pshipton in the reviews as early as possible.